lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/batch/AzureBatchHelper.java

```diff
@@ -136,11 +151,17 @@ public void submitTask(final String jobId, final String taskId, final URI jobJar
 
     LOG.log(Level.INFO, "Evaluator task command: " + command);
```

Comment:
use string interpolation

Reply:
Fixed.

---
In reply to: [207101845](https://github.com/apache/reef/pull/1478#discussion_r207101845) [](ancestors = 207101845)